pkg/pravega/bookie.go

```diff
@@ -211,3 +245,31 @@ func makeBookieConfigMap(pravegaCluster *v1alpha1.PravegaCluster) *corev1.Config
 		Data: configData,
 	}
 }
+
+func makeBookiePodDisruptionBudget(pravegaCluster *v1alpha1.PravegaCluster) *policyv1beta1.PodDisruptionBudget {
+	minAvailable := intstr.FromInt(3)
```

Comment:
This implies if the bookie count is `x, where x > 3` then `x-3` bookies can go down at the same time, right?
Also, should the behavior change based on `auto_recovery:  true/false` ?

Reply:
> This implies if the bookie count is x, where x > 3 then x-3 bookies can go down at the same time, right?

Correct. There will always be at least 3 bookies (planned disruptions). Does it sound reasonable to you from a data protection standpoint?

> Also, should the behavior change based on auto_recovery: true/false ?

Good point, but I'm not sure if it makes sense to make the distinction... Do you have any insight or suggestion?